web/src/lib/Movie/messages.ts

```diff
@@ -0,0 +1,30 @@
+import { type ToastType } from '../../ambient.d';
+
+const messageCommonProps = {
+  dismissible: true,
+  timeout: 3000
+};
+
+export const watchedError: ToastType = {
+  message: 'There was an error updating the movie',
+  type: 'error',
+  ...messageCommonProps
+};
+
+export const watchedSuccess: ToastType = {
+  message: 'Movie successfully updated',
```

Comment:
Maybe something like `Movie marked as watched`? Updated on the FE to me feels like you're updating something about the movie.

Reply:
I thought it was appropriate because we're updating the watched status of the movie? I was trying to avoid having 2 separate messages: "Movie marked as watched" vs "Movie marked as unwatched", or something like that.